test/benchmark/README.md

```diff
@@ -1,172 +1,280 @@
+}
+
+// sqlite/ext2_benchmarks/bench_results/ext2_updates_between.json
+{
+    "result_extraction": {
+        "search_pattern": "10000 UPDATES, numeric BETWEEN, indexed....",
+        "result_index": 8
+    },
+    "chart": {
+    ...
+}
+```
+
+### Step 3: Update Suite's `summary.json`
+
+Asterinas is an increasingly continuous improvement project. Consequently, while some benchmarks have been incorporated into the Benchmark Collection, their optimization is still ongoing. We do not wish to display these benchmarks on the overview charts. Therefore, we define the benchmarks that should be shown in the `summary.json` file. Only the benchmarks in the `summary.json` file can be displayed on the overview charts. Note that the standalone benchmark results are still available in the respective benchmark suite's page.
+
+To include a new benchmark in the suite's summary table, we need to update the `summary.json` file at the root of the suite. Taking `sqlite` for example:
+
+```jsonc
+// sqlite/summary.json
+{
+    "benchmarks": [
+        "sqlite/ext2_deletes_between",
+        "sqlite/ext2_deletes_individual",
+        "sqlite/ext2_refill_replace",
+        "sqlite/ext2_selects_ipk"
+    ]
+}
+```
+
+### Step 4: Update the CI Configuration
+
+Asterinas employs GitHub Actions for continuous integration (CI) to automatically execute benchmark collection every day. To incorporate the new benchmark into the CI pipeline, it is necessary to update `<bench_suite>/<bench_job>` within the `.github/benchmarks.yml` file.
+
+```yaml
+strategy:
+    matrix:
+    benchmarks:
+        - redis/ping_inline_100k_conc20_rps
+        - sqlite/ext2_benchmarks
+        ...
+```
+
+### Step 5: Test, Validate and Commit
+
+Before committing the changes, it is essential to test the new benchmark job locally to ensure it runs correctly. This step helps identify any issues or errors that may arise during the benchmark execution. 
+
+Firstly, we can run the benchmark locally to ensure it works as expected. The following command should finally generate the `result_<bench_suite>-<bench_job>.json` under `asterinas/`. 
+
+```bash
+cd asterinas/
+bash test/benchmark/bench_linux_and_aster.sh <bench_suite>/<bench_job>
+```
+
+Secondly, we can validate modifications by running the CI pipeline on our own repository. To do this, we need to modify the `runs-on` field from `self-hosted` to `ubuntu-latest` on `.github/benchmarks.yml`. Then, we can manually trigger the CI pipeline on our own repository to ensure the new benchmark is correctly executed. After validation, we can reverse the `runs-on` field back to `self-hosted`.
+
+Finally, if the new benchmark job runs successfully, we can commit the changes and create a pull request to merge the new benchmark into the main branch.
+
+## The `bench_result.json` Format
+
+The `bench_result.json` file configures how benchmark results are processed and displayed. Below is a concise description of its structure:
+
+### Alert Configuration
```

Comment:
Giving the descriptions of the fields in their own sections is ok. But the readers still desire to learn the big picture at first. So we should give an example of the file upfront. And I think it is sufficient to provide all the descriptions of the fields in the json file example (although json does not support comments...). I don't have a strong opinion on this matter.

Reply:
Yes, the current hierarchy is indeed intuitive enough using JSON. BTW, `jsonc (JSON with Comments)` supports comments.

Reply:
> BTW, jsonc (JSON with Comments) supports comments.

Great. I didn't know that before!